app/posts/2021-09-01-equality-diversity.md

```diff
@@ -0,0 +1,54 @@
+    href: https://ico.org.uk/media/1061/anonymisation-code.pdf
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+## Introduction
+
+When jobseekers apply for a job using the Teaching Vacancies application form, they are presented with optional diversity monitoring questions as part of the application process. When the application form is submitted, we ensure the diversity data is immediately separated from the application form, anonymised, and stored in a secure database. In user research, hiring staff from schools told us they would like to view this information to gauge how well their recruitment strategy is performing and ensure their workforce is diverse and inclusive.
+
+###  What we wanted to learn
+
+We wanted to agree how we can share diversity monitoring information from job applications with schools in a way that is unbiased, provides value to users and GDPR compliant.
+
+###  Hypothesis
+
+Our initial plan was to share the data on the service with schools after the closing date of a job and only if there were 5 or more applications for the job to reduce the risk of identifying applicants. We made this decision based on guidance from policy colleagues and the Information Commissioner’s Office.
+
+However, not all jobs will receive 5 or more applications, in which case schools will not be able to see the data, and even with the minimum number of 5 applications there are still some scenarios that could lead to identification of an applicant. This meant that in a lot of cases, this would not be a useful way to share the information.
+
+{{ appFigure({
+  image: {
+    file: "1.0.9 - Equal opportunities data.png",
+    alt: "A screenshot of a proposed design for sharing equality and diversity data with hiring staff."
+  }
+}) }}
+
+## What we did
+
+We sought advice from the Legal Advisers Office and received a response in writing informing us that:
+
+* Teaching Vacancies are not obligated to share the equality & diversity data but we can if we want to (internally within the department and with schools).
+* If we do decide to share the data, we need to be clear at the point of input from the applicant so that they know the data will or could be shared and ensure we let them know what we do to protect their information (how we anonymise their data).
+* There is no timescale on sharing the data, so we can share this information how and when we feel is reasonable (for example, in real time or once a quarter).
+* We must abide by GDPR.
+
+## What did we decide
+
+We held a workshop to agree within the team how we might do this and came to the conclusion that it was too soon to develop a way to share diversity data. We first need a better understanding of the following:
+
+* How do schools use diversity data? (breaking this down into types of school, types of hiring staff users within schools)
+* Who in the department is interested in using the data? What would they use it for?
```

Comment:
I think all the questions could be bulleted rather than having 2 questions in one bullet? It just  looks a bit weird that the last one land on a new line and doesn't have a bullet point